2-mandatory/1-weather-report.js

```diff
@@ -12,6 +12,11 @@
 */
 
 function getTemperatureReport(cities) {
+    const arrayOfString = [];
+         for (let city of cities) {
+            arrayOfString.push(`The temperature in ${city} is ${temperatureService(city)} degrees`)         
+         }
+         return arrayOfString;
```

Comment:
👏

Reply:
I like how you have simplified your code Melese. I found it easier to understand.